docs/usage/openapi/index.rst

```diff
@@ -6,15 +6,19 @@ Litestar has first class OpenAPI support offering the following features:
 - Automatic `OpenAPI 3.1.0 Schema <https://spec.openapis.org/oas/v3.1.0>`_ generation, which is available as both YAML
   and JSON.
 - Builtin support for static documentation site generation using several different libraries.
-- Simple configuration using pydantic based classes.
+- Full configuration using pre-defined type-safe dataclasses.
 
 
 Litestar includes a complete implementation of the `latest version of the OpenAPI specification <https://spec.openapis.org/oas/latest.html>`_
-using Python dataclasses. This implementation is used as a basis for generating OpenAPI specs, supporting builtins including
-``dataclasses`` and ``TypedDict``, as well as Pydantic models and any 3rd party entities for which a plugin is implemented.
+using Python dataclasses. This implementation is used as a basis for generating OpenAPI specs,
+supporting :class:`~dataclasses.dataclass`, :class:`~typing.TypedDict`,
+as well as Pydantic and msgspec models, and any 3rd party entities
+for which a :ref:`plugin <plugins>` is implemented.
 
 This is also highly configurable - and users can customize the OpenAPI spec in a variety of ways - ranging from passing
-configuration globally, to settings specific kwargs on route handler decorators.
+configuration globally to setting
+:ref:`specific kwargs on route <openapi/schema_generation:Configuring schema generation on a route handler>`
```

Comment:
```suggestion
:ref:`specific kwargs on route <usage/openapi/schema_generation:Configuring schema generation on a route handler>`
```

Reply:
I am not so sure on this one, but if you want to try, you can accept this suggestion and check 🙂